### PR TITLE
updates cargo.toml file & readme

### PR DESCRIPTION
**Pull request by jpvajda**
* Updates Cargo.toml file to point to new repo location
* Updates readme to be more consistent with DX standards

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1208129830345857

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -18,15 +24,21 @@ bytes = "1"
...
 serde = { version = "1.0.25", features = ["derive"] }
 serde_json = "1"
 serde_urlencoded = "0.7"
 thiserror = "1"
 tokio = { version = "1.38.0", features = ["full"] }
 tokio-stream = "0.1.15"
-tokio-tungstenite = { version = "0.20.1", features = ["rustls-tls-webpki-roots"], optional = true }
+tokio-tungstenite = { version = "0.20.1", features = [
+  "rustls-tls-webpki-roots",
+], optional = true }
```
Comment (jcdyer) [inline Cargo.toml:41]:
```suggestion
tokio-tungstenite = { 
  version = "0.20.1",
  features = ["rustls-tls-webpki-roots"], 
  optional = true
}
```

Comment (jcdyer) [reply Cargo.toml:41]: This looks a bit nicer to me than splitting the features list onto multiple lines but inlining the arguments

File: README.md [hunk 2]
```diff
@@ -1,25 +1,29 @@
 # Deepgram Rust SDK
 
+[![Discord](https://dcbadge.vercel.app/api/server/xWRaCDBtW4?style=flat)](https://discord.gg/xWRaCDBtW4)
 [![CI](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml/badge.svg?branch=main)](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml)
```
Comment (jcdyer) [inline README.md:4]: This URL should no longer be deepgram-devs

**Review by jcdyer — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (jpvajda) [reply README.md:4]: Good catch. will fix.

**Review by jcdyer — APPROVED** [review]

**Pull request merged**